physics/docs/pdftxt/GFSv15p2_suite.txt

```diff
@@ -26,7 +26,7 @@ The GFS_v15p2 physics suite uses the parameterizations in the following order:
 \code
 <?xml version="1.0" encoding="UTF-8"?>
 
-<suite name="FV3_GFS_v15p2" lib="ccppphys" ver="5">
```

Comment:
Please replace `lib="ccppphys" ver="1"` with just `version="1"` to be consistent with the new syntax.